cloud/scaling.rst

```diff
@@ -0,0 +1,34 @@
+Scaling
+#######
+
+Citus Cloud provides self-service scaling to deal with increased load. The web interface makes it easy to either add new worker nodes or increase existing nodes' memory capacity.
+
+For most cases either approach to scaling is fine and will improve performance. However there are times you may want to choose one over the other. If the cluster is reaching its disk limit then adding more nodes is the best choice. Alternately, if there is still a lot of headroom on disk but performance is suffering, then scaling node RAM is the way to go.
+
+Both adjustments are available in the formation configuration panel of the settings tab:
+
+.. image:: ../images/cloud-formation-configuration.png
+
+Clicking the node change link provides two sliders:
+
+.. image:: ../images/cloud-nodes-slider.png
+
+After you adjust the sliders and accept the changes, Citus Cloud begins applying the changes subject to the user-specified maintenance window:
+
+.. image:: ../images/cloud-maintenance-window.png
+
+Resizing nodes creates a PostgreSQL follower for each node, where the followers have the new hardware instance type. Creating follower nodes uses the same replication mechanism that powers Cloud's high-availability feature. Once the resize is initiated, it takes an average of forty minutes per hundred gigabytes of data for the follower's data to be fully synchronized. After the synchronization is complete, Cloud does a quick failover from the existing primary nodes to their followers which takes about one minute.
```

Comment:
@sumedhpathak this paragraph contains the same sentiment as was removed in https://github.com/citusdata/citus_docs/pull/330/commits/8f4890f43583460d2424c3047a2b69c3c33bd008 about reusing "HA plumbing" for instance upgrades. I can expand on it more if you like.